src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  person: Person,
+}
+
+export const PersonInfo: FC<Props> = (props) => {
+  const { person } = props;
+  const {
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+    slug,
+  } = person;
+
+  const { userSlug } = useParams();
```

Comment:
The useParams hook is missing its type. Consider specifying the type for the useParams hook like this: `useParams<{ userSlug: string }>()`